src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqTouchstoneRepository.kt

```diff
@@ -133,7 +133,7 @@ class JooqTouchstoneRepository(private val scenarioRepository: () -> JooqScenari
     fun mapCoverageRow(record: Record, scenarioDescriptionId: String) = CoverageRow(
             scenarioDescriptionId,
             record[COVERAGE_SET.ID],
-            record[SCENARIO_COVERAGE_SET.ORDER],
+            record[SCENARIO_COVERAGE_SET.ORDER] + 1,
```

Comment:
It might be nice to have a comment here, explaining why we are adding one to this value. Although, now I think about that, did I even explain that to you? The issue here is that CSV file is going to be consumed by academics, who are in general not programmers, and not familiar with zero indexing. This is just to make things a bit more human friendly.